recipes-soletta/soletta/soletta_git.bb

```diff
@@ -29,6 +29,9 @@ PACKAGES = " \
          ${PN} \
 "
 
+PACKAGECONFIG ??= "x11"
+PACKAGECONFIG[x11] = ",,cairo atk gtk+3 gdk-pixbuf pango,"
```

Comment:
okay, but we should create a split package for the gtk modules... otherwise if Ostro is built with x11 support, using soletta would always pull in the gtk module and all the dependencies!

@brunobottazzini can you look into a split package for this? Like soletta-module-flow-gtk, it would be including only gtk.so and associated json. Just be aware to test when `x11` is disabled since it would produce an empty package, which leads bitbake to fail. (I solved this in our systemd package, I guess with a flag to allow empty packages)